software/soapysdr-xtrx/XTRXDevice.cpp

```diff
@@ -102,6 +102,11 @@ SoapyXTRX::SoapyXTRX(const SoapySDR::Kwargs &args)
         0 * (1 << CSR_LMS7002M_CONTROL_TX_RX_LOOPBACK_ENABLE_OFFSET)
     );
 
+    //Enable DMA Synchronizer
+    #ifdef CSR_PCIE_DMA0_SYNCHRONIZER_ENABLE_ADDR
+    litepcie_writel(_fd, CSR_PCIE_DMA0_SYNCHRONIZER_ENABLE_ADDR, 0b10);
+    #endif
```

Comment:
Why the `#ifdef` guard?

Reply:
When I was using this branch as a base for the peripheral changes: https://github.com/JuliaComputing/xtrx_julia/pull/104/commits/b6a668ebf61ac8545f44481125efa502b50b49e9

Reply:
Yingbo and Chris Elrod turned me on to Clangd. I will get this in that PR with those lint fixes.